python/run-tests.sh

```diff
@@ -93,11 +93,19 @@ else
   noseOptionsArr="$(find "$targetDir" -type f | grep "\.py" | grep -v "\.pyc" | grep -v "\.py~" | grep -v "__init__.py")"
 fi
 
+# Limit TensorFlow error message
+# https://github.com/tensorflow/tensorflow/issues/1258
```

Comment:
I am not quite certain whether it is due to `--nologcapture` or that plus this that suppressed the TensorFlow INFO logs. 

For tests, I believe they did run
https://travis-ci.org/databricks/spark-deep-learning#L1099